lib/cocoapods/sandbox/path_list.rb

```diff
@@ -52,9 +52,18 @@ def read_file_system
         end
         escaped_root = escape_path_for_glob(root)
 
-        absolute_paths = Pathname.glob(escaped_root + '**/*', File::FNM_DOTMATCH).lazy
-        dirs_and_files = absolute_paths.reject { |path| path.basename.to_s =~ /^\.\.?$/ }
-        dirs, files = dirs_and_files.partition { |path| File.directory?(path) }
+        absolute_paths = Pathname.glob(escaped_root + '**/*', File::FNM_DOTMATCH)
+        absolute_paths = absolute_paths.reject { |path| path.basename.to_s =~ /^\.\.?$/ }.sort_by(&:to_s)
```

Comment:
any reason for the `sort_by`?

Reply:
Just checked this and the `.lazy` and it seems to work without it - this may have been something else I was trying and forgot to remove, so I'll remove it